src/alr/alr-commands-toolchain.ads

```diff
@@ -32,12 +34,13 @@ package Alr.Commands.Toolchain is
           & "select the default toolchain for this configuration. "
           & "Adding --local will instead make the selection apply "
           & "only to the workspace (overridding a possible "
-          & "configuration-wide selection). Giving a release argument will "
-          & "skip the assistant and set the release as the default.")
+          & "configuration-wide selection). Giving one or more release"
```

Comment:
Sorry for the nitpicking about this, but the way that sounds ok to me is "one or more releases".

Reply:
That's what reviews are for :)